tests/specs/test-customer-sync.php

```diff
@@ -121,11 +123,11 @@ function test_customer_sync_validation() {
 		$should_sync = $record->should_sync();
 		$this->assertFalse( $should_sync );
 
-		// test no name
+		// test no name, falls back to username and should still sync
 		$record = new TaxJar_Customer_Record( $customer->get_id(), true );
 		$record->load_object();
 		$should_sync = $record->should_sync();
```

Comment:
Test has been updated to now confirm that even if a first and last name is not present on the customer, the customer will still sync to TaxJar (due to the fallback to username).